aws-models/accessanalyzer.json

```diff
@@ -1,6 +1,24 @@
 {
   "smithy": "2.0",
   "shapes": {
+    "com.amazonaws.accessanalyzer#Access": {
```

Comment:
I notice we are downloading the updated models but I can't remember if I was ever using them. I think I am still using the AWS SDK v2 to get the types (a lazy approach) but wanted to generate them from the models.

Reply:
Oh, I have no idea either. Will try to have a look soon.